index.bs

```diff
@@ -424,18 +455,52 @@ For each <a>element proxy</a> for that element, perform the following steps:
     7. For each property in |inputProperties|, if the property’s computed value has changed,
         set the <a>animation requested flag</a> on the |animator| to <a>frame-requested</a>.
 
-The timelines present in an animator's <a>animator timeline list</a> can also affect the
-<a>animation requested flag</a>:
+Animator specifies a DocumentTimeline {#requesting-animation-frames-document-timeline}
+-------------------------------------
+
+If a {{DocumentTimelineOptions}} instance is present in an animator's <a>animator timeline list</a>
+then that animator's <a>animation requested flag</a> <em>must</em> be set to <a>frame-requested</a>
+at the start of each animation frame.
+
+Animator specifies a ScrollTimeline {#requesting-animation-frames-scroll-timeline}
+-----------------------------------
+
+If a {{ScrollTimelineOptions}} instance is present in an animator's <a>animator timeline list</a>
+then the user agent <em>must</em> run the following steps when the scroll position of the nearest
+non-visible overflow ancestor to the animator’s <a>root element</a> changes:
+
+  1. For each {{ScrollTimelineOptions}} instance |options| in the animator's <a>animator timeline
+       list</a> do the following steps:
+
+    1. Let |orientation| be the result of <a>Get</a>(|options|, "orientation").
+
+    2. Let |startScrollOffset| be the result of <a>Get</a>(|options|, "startScrollOffset").
+
+    3. Let |endScrollOffset| be the result of <a>Get</a>(|options|, "endScrollOffset").
+
+    4. Let |fillMode| be the result of <a>Get</a>(|options|, "fill").
+
+    5. Process the |options| according to the first matching conditional in the following:
+
+         <div class="switch">
+
+         :  If <em>any</em> of the following are true:
+
+             1. the orientation of the change in scroll position does not match |orientation|, or
+
+             2. the scroll position is before |startScrollOffset| and |fillMode| is <b>none</b> or
+                  <b>backwards</b>, or
+
+             3. the scroll position is after |endScrollOffset| and |fillMode| is <b>none</b> or
+                  <b>forwards</b>.
+
+         :: do not change the animator's <a>animation requested flag</a> value.
+
+         :  Otherwise
+         :: set the animator's <a>animation requested flag</a> to <a>frame-requested</a>.
```

Comment:
Can we not defer to the scroll animations spec and just say that if the currentTIme output of the scroll animation has changed then we set the requested flag to frame-requested?